library/Fission/CLI/Environment/Override/Types.hs

```diff
@@ -48,7 +47,7 @@ instance Monoid Override where
 
 instance ToJSON Override where
   toJSON Override {..} = object $ catMaybes
-    [ ("peers"      .=) <$> tailMaybe peers
```

Comment:
I really can't figure out why this would've been written like this in the first place? It seems like extra work :thinking: